packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -191,13 +214,16 @@ class Paginator extends React.Component {
       intl,
       totalCount,
       itemCountPerPage,
+      paginatorHeadingId,
+      paginatorLabel,
     } = this.props;
     const totalPages = calculatePages(totalCount, itemCountPerPage);
     const previousPageIndex = selectedPage === 1 ? 1 : selectedPage - 1;
     const nextPageIndex = selectedPage === totalPages ? totalPages : selectedPage + 1;
+    const paginatorAriaLabel = paginatorLabel || 'pagination';
 
-    const reducedView = (
-      <div className={cx('paginator', !totalCount && 'pageless', theme.className)} role="navigation" aria-label="pagination">
+    const reducedViewDetail = (
```

Comment:
Suggested name for clarity of usage
```suggestion
    const reducedViewChildren = (
```